09-SyOps.Rmd

```diff
@@ -1,58 +1,52 @@
 
-- Data Owners are ultimately responsible for who has access to data while in the platform.
-- Data Owners **must** actively manage delegated access for team members (popularly - Joiners, Movers, Leavers)
-- Self-managed permissions through [data access groups](https://moj-analytical-services.github.io/platform_user_guidance/user-administration.html#teams-and-data-access-groups) must be used to provision and remove access as suitable - which is solely defined by Data Owners.
+__App admins and data source admins will:__
 
-## Data Minimisation
+* ensure that app and data source users have the correct read/write permissions;
+* ensure that app and data source users only have access to the minimum data required for them to perform their job; and
+* regularly review access permissions for app and data source users, including when users join or leave MoJ, or move within MoJ.
 
-- Consider [minimising data](https://moj-analytical-services.github.io/platform_user_guidance/annexes.html#data-minimisation) through pseudonymisation and anonymisation techniques.
+## GitHub
 
-## Guidance on the use of Github
+GitHub can be used to store code with a security marking of OFFICIAL only. Where possible, you should store data outside of GitHub even when it is not covered by the below restrictions.
 
-The advice in this document relates to private repositories which are part of the [MoJ Analytical Services organisation](https://github.com/moj-analytical-services/) only. **Do not store any work in public repositories** unless you are sure it is not sensitive and can be made public. Have prior permission from your line manager before releasing to a public repository. Read the [Service Manual pages on opensource](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable).
+__All users will:__
 
-### What can I use GitHub for?
-
-When you have approval, you may use GitHub to store your analytical **code** _but not data_ at the `OFFICIAL` level. This means that you can use Git for work where the code itself is sensitive, such as work on court or prison closures.
-
-You may also use GitHub to store project writeups. However, the write ups must not contain information or data which would violate the prohibitions on the use of GitHub set out below.
-
-### Prohibitions: What are the limits on GitHub use?
-
-In general, you must not use GitHub to store datasets. Specifically, you **MUST NOT** use GitHub for the following purposes:
-
-* To store any large datasets classified at `OFFICIAL` or above. Our working definition is that a large dataset contains over 1,000 records.
-
-* To store any data whatsoever (in code or in datasets) where individuals are potentially identifiable and the data is not already in the public domain.
-
-* To store credentials such as usernames and passwords, or secrets such as database connection strings.
-
-## Reporting security incidents
```

Comment:
Is this being moved somewhere else?

Reply:
I've moved this to the information governance chapter [here](https://github.com/moj-analytical-services/platform_user_guidance/pull/55/files/3571c9b5ec25e9563becd46768bd88e37c7e8f92#diff-f5cd72837246666489e9bca165a4ac73R188]. I've linked to the guidance on the intranet instead of just including an email address.

Reply:
👍